### ISSUE TITLE
Aria-required property is missing for required field in Editor screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#878]
**Describe the bug**
Aria-required property is missing for required field.

**To Reproduce**
1. Go to Editor screen and select any page for a site and edit it.
2. Go to "Actions" and click on "Edit Metadata". A pop-up opens. Expand "Meta-data".
3. "File name" is a required field. Inspect the "File name" text box element, you will find that the Aria-required=true property is missing for the input text.

**Expected behavior**
Aria-required=true property must be available for all required fields.

**Additional context**
Navigate all the options available for similar issue and fix.
